storio/src/androidTest/java/com/pushtorefresh/storio/db/integration_test/impl/BaseTest.java

```diff
@@ -102,4 +111,19 @@
 
         return deleteResult;
     }
+
+    void checkNotEmptyAndEquals(@Nullable final List<User> first,
+                                @Nullable final List<User> second) {
+        assertNotNull(first);
+        assertNotNull(second);
+        assertEquals(first.size(), second.size());
+
+        for (User userFromFirst : first) {
+            assertTrue(second.contains(userFromFirst));
```

Comment:
This check is not correct, it checks that lists contain same items, but order of items can be different. I remember that I wrote own check for comparing lists, but may be better to use `assertEquals(firstList, secondList)`? Standard `List` implementations have correctly overridden `equals()`